cats/CI_api_query.py

```diff
@@ -41,12 +37,14 @@ def get_tuple(
     session = requests_cache.CachedSession('cats_cache', use_temp=True)
     # get the carbon intensity api data
 
-    r = session.get(request_url(dt, postcode))
+    API = API_interfaces["carbonintensity.org.uk"] # TODO give choice of API to user
+
+    r = session.get(API.get_request_url(dt, postcode))
     data = r.json()
 
-    return parse_data_from_json(data)
+    return API.parse_response_data(data)
```

Comment:
This function wasn't renamed in [the definition of `APIInterface`](https://github.com/GreenScheduler/cats/blob/76d61e6f141e7e96b5861b0510364690b54943ca/cats/CI_api_interface.py#L7) though

Reply:
I'm not sure I follow, `APIInterface` is still using `parse_response_data`:
```python
APIInterface = namedtuple('APIInterface', ['get_request_url', 'parse_response_data'])
```

Reply:
I'm not sure I follow, `APIInterface` is still using `parse_response_data`:
```python
APIInterface = namedtuple('APIInterface', ['get_request_url', 'parse_response_data'])
```

Reply:
Ah sorry - my mistake! All good :)